createdb.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# TODO - it'll be trivial, but put this into a helper perhaps config.go and
+# call it any time we try to open the db and find it doesn't exist.
+# For now, run this script by hand to create the db
+db=/var/lib/lxd/lxd.db
+if [ $# -eq 1 ]; then
+	db=$1
+fi
+rm -f $db
+cat << EOF | sqlite3 $db
+create table certificates (
+id integer primary key autoincrement not null,
+fingerprint varchar(255) not null,
+type integer,
+name varchar(255) not null,
+certificate blob not null,
+unique (fingerprint)
+);
```

Comment:
Just a small nitpick and probably irrelevant since that script will go away, but I tend to appreciate a nicer SQL formatting :)

For the case of this table, that'd be something like:

    CREATE TABLE certificates {
        id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
        fingerprint VARCHAR(255) NOT NULL,
        type INTEGER,
        name VARCHAR(255) NOT NULL,
        certificate BLOB NOT NULL,
        UNIQUE (fingerprint)
    );